packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
I'm fine with just the callback mode, but if the point is to get the results of the operation to the user as quickly as possible, event emitters can emit status updates as they happen. If you think of streams as for reading and writing, as far as I can tell, we aren't using this for either-- we're just using it to let the user know the results of their request, that happened to be sent as a stream. This use case seems better suited for an event emitter than a readable stream.